bentoml/_internal/runner/runner.py

```diff
@@ -247,9 +247,13 @@ def scheduled_worker_count(self) -> int:
             self.resource_config,
         )
 
-    def setup_worker(self, worker_id: int) -> None:
-        self.scheduling_strategy.setup_worker(
-            self.runnable_class,
-            self.resource_config,
-            worker_id,
-        )
+    @property
+    def scheduled_worker_env_map(self) -> dict[int, dict[str, t.Any]]:
```

Comment:
does Yatai need this information for scheduling runners?

Reply:
the worker concept is transparent for yatai

Reply:
@bojiang got it, in the case of Yatai, it will just use resources available from system.

Reply:
yeah